### ISSUE TITLE
Authentication fail

### ISSUE THREAD

**Issue by Rajshah5472** [ChensoftDev/MiniProject#1]
when i try to regitser new user everytime show me authentication fail

**Comment by ChensoftDev** [comment]
Server is down mate, it is old project

**Comment by Rajshah5472** [comment]
sir i delete your server and try to connect new one still i am facing same error if you how to connect with new server please let me know
and added SHA aswell but stil geting same error

**Comment by ChensoftDev** [comment]
Are you unitec student?

**Comment by Rajshah5472** [comment]
yes dont worry i am not copying your assignment i just want to check how its running

**Comment by ChensoftDev** [comment]
Good luck

**Comment by Rajshah5472** [comment]
can you help me

**Comment by ChensoftDev** [comment]
Try to do by yourself bro

**Comment by Rajshah5472** [comment]
bro can you help to do authentication

**Comment by Rajshah5472** [comment]
bro please please help me in this assigment i am happy to pay if you need
